src/test/java/com/epam/ui/BaseTest.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.ui;
+
+import com.codeborne.selenide.testng.ScreenShooter;
+import com.epam.page.FilterPage;
+import com.epam.page.WelcomePage;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+import static com.codeborne.selenide.Selenide.closeWebDriver;
+import static com.codeborne.selenide.Selenide.open;
+import static com.codeborne.selenide.WebDriverRunner.getWebDriver;
+import static com.epam.constant.Constants.FILTERS_PAGE_URL;
+import static com.epam.constant.Constants.SIGNED_IN_NOTIFICATION;
+
+public class BaseTest {
+    private final Logger logger = LogManager.getLogger(BaseTest.class);
+    private final WelcomePage welcomePage = new WelcomePage();
+    private final FilterPage filterPage = new FilterPage();
+
+    @BeforeTest
```

Comment:
with such approach all the test methods will be dependant on each other, and as we can see the design tells us to execute tests in strict order which is against of the Best Practice

Reply:
My bad: I decided to cover e2e flow. Will update according to the Best Practice